lib/roby/robot.rb

```diff
@@ -50,7 +50,7 @@ def self.method_missing(name, *args)
 	end
 
 	options = args.first || {}
-	task, planner = Roby.app.prepare_action(name, **options)
+	task, planner = Roby.app.prepare_action(name, job_id: Roby::Interface::Job::allocate_job_id, **options)
```

Comment:
I'll be closing this PR, I have this implemented with unit tests already ...

Just one comment for your information: even though it is possible to use `::` to call module or class-level methods in Ruby, it's encouraged to stick with the dot, e.g.

```
Roby::Interface::Job.allocate_job_id
```